src/DGtal/dec/doc/moduleDECIntroduction.dox

```diff
@@ -219,12 +222,12 @@ For obvious reasons, the ambient dimension must be equal or greater than the emb
 The ambient dimension is used internally by DiscreteExteriorCalculus.myKSpace and therefore is the dimension of cells and points used to fill the DEC object.
 The third template parameter is a linear algebra backend used internally to specify containers and solvers.
 By now, there is only one available backend: EigenLinearAlgebraBackend.
-For example, here is a snippet that define a working two dimensional DEC object embedded in a two dimensional ambient space.
+For example, here is a snippet that define a working two dimensional DEC object embedded in a two dimensional ambient space along with the corresponding factory type.
```

Comment:
Just a comment: on DOX files, could you please justify the text to 80-col to make it easier to read in github/terminal?